### PR TITLE
Reference M.TP.P.WinRT to get Windows.winmd references for S.R.WR ref

### PR DESCRIPTION
**Pull request by weshaggard**
cc @ericstj

### CODE REVIEW

File: src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj [hunk 1]
```diff
@@ -23,10 +23,6 @@
     <Compile Include="System.Runtime.WindowsRuntime.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferencePath Include="$(ExternalAPIsPath)\NetFX\Contracts\DesignTimeFacades\mscorlib.dll" />
```
Comment (ericstj) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: You didn't need mscorlib?  I thought the projections emitted refs to mscorlib, not System.Runtime.

Comment (weshaggard) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: it gets added automatically by our framework targeting targets.

Comment (ericstj) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: Not for reference assemblies.

Comment (ericstj) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: Indeed, if I build with the dev12 compiler I get
```
  System.Runtime.WindowsRuntime.cs(50,85): error CS0012: The type 'System.Enum'
 is defined in an assembly that is not referenced. You must add a reference to
assembly 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c5619
34e089'. [Z:\src\corefx\src\System.Runtime.WindowsRuntime\ref\System.Runtime.Wi
ndowsRuntime.csproj]
```

Comment (weshaggard) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: Interesting you are right it isn't referenced. Why is that error not showing up with Dev14 toolset?

Comment (ericstj) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: I think roslyn is less aggressive about walking type refs (in some cases, more in others).

Comment (weshaggard) [inline src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj]: Interesting. I suspect the metadata reader doesn't need to go through mscorlib references for these primitive types. That is nice, I didn't realize that was the case. At any rate I added Microsoft.NETCore.Portable.Compatibility package reference to get the mscorlib dependency.

**Comment by weshaggard** [comment]
@dotnet-bot test Innerloop Ubuntu Debug Build and Test

**Comment by ericstj** [comment]
LGTM.

**Pull request merged**